### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_AzureSQLMigration_DeveloperTool:The role for refresh button is defined as a link.

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#15854]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG4.1.2;#AINotInScope;#DeveloperTool

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Refresh button .
5. open developer tool using ctrl+shift+I and hover over to the button.

### Actual:
The role for refresh button is defined as a link.

### Expected:
The role should be defined as a button.

### User Impact:
Screen reader users will get wrong information about the refresh button.

**Comment by aasimkhan30** [comment]
Change the role of the link to the button.

**Comment by Harika1702** [comment]
![15854_Issue Resolved](https://user-images.githubusercontent.com/100270074/218437516-46df3b1e-a278-4b7f-8871-e39cee81ac16.jpg)
Verified this issue in the below environment.
Version: 1.42.0-insider (user setup)
Commit: 99a924dbcdce7bbaa0ab78b19f84e83035e54651
Date: 2023-02-08T06:58:10.882Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22623
GitHub Tags: #Closed;
